### DISCUSSION TITLE
An error occurs when executing a docker in version 2.7: ps not found

### DISCUSSION THREAD

**Discussion by pjy1368** [dadoonet/fscrawler#1259] [Q&A] [answered]
Hi. I follow the official guide document for installing the fscrawler using a docker, but the following error occurred.

```
/usr/bin/fscrawler: 47: /usr/bin/fscrawler: ps: not found
ERROR StatusLogger Reconfiguration failed: No configuration found for '55054057' at 'null' in 'null'
```

In addition, no work is performed and unlimited loading is performed.
My folder structure is as follows.

![image](https://user-images.githubusercontent.com/56083021/134435393-ed368783-6f96-46d8-b1d8-4272613c020a.png)

The folder '.fscrawler' is an empty folder, and an example document file exists in the folder 'tmp'.

**Comment by dadoonet** [comment, marked as answer]
It's a bug. See https://github.com/dadoonet/fscrawler/issues/1229
